### PR TITLE
Transform package names for case insensitive ecosystems

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
We would probably like to utilize this with functions like `MavenCoordinates.normalize_str()` to centralize sanity checks and transformations on package names that enter to our system.

File: cucoslib/workers/init.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('version'))
         self._strict_assert(arguments.get('ecosystem'))
 
+        # make sure we store package name based on ecosystem package naming case sensitivity
+        arguments['name'] = case_sensitivity_transform(arguments['name'])
```
Comment (tuxdna) [inline cucoslib/workers/init.py]: `case_sensitivity_transform` takes two arguments `(ecosystem, name)` , but here only one argument is provided `arguments['name']`. Is this correct ?

Comment (fridex) [inline cucoslib/workers/init.py]: Thanks! Nice catch. I was too fast with `git push` :)

**Comment by fridex** [comment]
PTAL @tuxdna

**Comment by tuxdna** [comment]
LGTM, merging now.

**Pull request merged**